packages/manager/modules/cloud-connect/src/template.html

```diff
@@ -0,0 +1,155 @@
+<oui-header
+    data-heading="{{ :: 'cloud_connect_title' | translate }}"
+></oui-header>
+
+<oui-tile
+    data-heading="{{:: 'cloud_connect_order_follow_up_title' | translate}}"
+    data-ng-if="$ctrl.orders.length > 0"
+>
+    <oui-tile-definition data-ng-repeat="order in $ctrl.orders track by $index">
+        <oui-tile-description class="oui-progress-tracker">
+            <div class="d-flex justify-content-between">
+                <span>
+                    <strong
+                        data-translate="cloud_connect_order_follow_up_order_id"
+                    ></strong>
+                    <a
+                        data-ng-href="{{ order.orderBillingUrl }}"
+                        rel="noopener"
```

Comment:
Ok for the `rel="noopener"` but not target is defined here? By default the link will be open in the current frame you click on the link

Reply:
Yes, it's what we want to do.